gp-templates/translation-row-editor.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * Template for the editor part of a single translation row in a translation set display
+ *
+ * @package    GlotPress
+ * @subpackage Templates
+ */
+
+/**
+ * Filter to update colspan of editor. Decrease to add an extra column
+ * with action 'gp_translation_row_editor_columns'.
+ *
+ * @since 3.0.0
+ *
+ * @param int $colspan The colspan of editor column.
+ */
+$colspan = apply_filters( 'gp_translation_row_editor_colspan', $can_approve ? 5 : 4 );
+
+$singular = sprintf(
+	/* translators: %s: Original singular form of the text */
+	__( 'Singular: %s', 'glotpress' ),
```

Comment:
You can't include the 'Singluar:' part of the string here otherwise when we call `prepare_original()` later on it will get included in the markup, which we don't want.  For example of "Singular" was defined in a glossary then the "Singluar" before the colon would get get highlighted.